### ISSUE TITLE
UserAgents is deprecated in AlgoliaSearchClient 4.8

### ISSUE THREAD

**Issue by devladinci** [algolia/instantsearch-core-swift#10]
## Issue

- Can not compile since userAgents is deprecated in AlgoliaSearchClient V4.8. InstantSearch tries to decorate with bundle info.

```private func updateClientUserAgents() {
        let bundleInfo = Bundle(for: type(of: self)).infoDictionary!
        let name = bundleInfo["CFBundleName"] as! String
        let version = bundleInfo["CFBundleShortVersionString"] as! String
        let libraryVersion = LibraryVersion(name: name, version: version)
        let client = index.client
        if client.userAgents.index(where: { $0 == libraryVersion }) == nil {
            client.userAgents.append(libraryVersion)
        }
    }```

```./Pods/InstantSearch-Core-Swift/Sources/Searcher.swift:246:20: Cannot use mutating member on immutable value: 'userAgents' is a get-only property```

**Comment by clement-leprovost** [comment]
@devladinci Oops, you're right! Thanks for reporting it. I forgot to update InstantSearch Core when updating the client... Sorry for that. 😬

It should already be fixed in the `master` branch (commit `19edf62`). Official release will follow soon.

**Comment by clement-leprovost** [comment]
[Version 1.0.1](https://github.com/algolia/instantsearch-core-swift/releases/tag/1.0.1) has just been released and solves this issue.
